lib/gst/player/gstplayer.c

```diff
@@ -112,6 +122,9 @@ struct _GstPlayerPrivate
   GstPlayerState app_state;
   gint buffering;
 
+  GstPlayerMediaInfo  *media_info;  /* current media info */
+  GstPlayerMediaInfo  *media_info_copy; /* last emited media info */
```

Comment:
If it is, then the application has to take another reference. Sorry that I didn't notice before that this was what caused the confusion :)